### ISSUE TITLE
Anothr Problem with Metamask on Firefox

### ISSUE THREAD

**Issue by ferdinand0101** [MetaMask/faq#112]
![wtf meta mask](https://user-images.githubusercontent.com/24229481/35876350-47f1ebc8-0b72-11e8-86e0-1d34c37bc200.PNG)

**Comment by ferdinand0101** [comment]
I don't get this

**Comment by ferdinand0101** [comment]
Apperntly it works with Chrome

**Comment by Altcoinbroker** [comment]
YeaH i used it with Opera using the chrome extension plugin and had a similar issue.  Can you post the transaction? One time I sent the wrong token to EOS and it was $300 roughly. They would not budge  or return my emails.. Needless to say I hacked them for 300k and then I got their attention. I have screen shots of the funds in my wallet. I did return the funds minus my $300 of coarse, because I guess im more ethical than a lot of people. Id just be hurting the crypto market and taking money from investors in the end.  It was still one of the hardest decisions Ive ever made in my life. Thats a lot of money $$$. Let us know how it goes and post the transaction so I can see and try to help you. One think Im good at is recovering funds. for myself and other people. Also I strongly recommend you get the HTTPS everywhere extension and if your not running tails or know what this is then you should def look into for optimal security and encryption.

**Comment by chronaeon** [comment]
@ferdinand0101 thanks for this bug report. How many times have you experienced this problem when trying to send?

**Comment by Altcoinbroker** [comment]
only once. np though its been gr8 ty

**Comment by ferdinand0101** [comment]
i experienced it 3 times in a row after That i tried chrome and it worked
